lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -149,21 +189,46 @@ def filter_recommended_courses(user, unfiltered_course_keys, recommendation_coun
             - marketing url for the course (marketing_url)
     """
     filtered_recommended_courses = []
-    fields = ["title", "owners", "marketing_url"]
+    course_data_dict = {}
```

Comment:
nit: we should avoid using data types in variable names
```suggestion
    recommended_course_data = {}
```